src/common/ceph_fs.cc

```diff
@@ -76,8 +76,14 @@ int ceph_flags_sys2wire(int flags)
        ceph_sys2wire(O_CREAT);
        ceph_sys2wire(O_EXCL);
        ceph_sys2wire(O_TRUNC);
+
+       #ifndef _WIN32
```

Comment:
please avoid unnecessary indent.

Reply:
This already merged as part of the previous PR (https://github.com/ceph/ceph/pull/32780/commits).